packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -83,6 +84,9 @@ class ReorderableListView extends StatefulWidget {
   /// The amount of space by which to inset the [children].
   final EdgeInsets padding;
 
+  /// When start Dragging, Control to Extend list by adding last item
```

Comment:
I'm not really sure what this means and it is not a complete sentence.  What is this actually doing?

All doc comments should start with a complete sentence ending with a `.`.